lib/code_corps/adapter/map_transformer.ex

```diff
@@ -0,0 +1,86 @@
+defmodule CodeCorps.Adapter.MapTransformer do
```

Comment:
I did some testing, but generally, the stripe adapter layer is already well-tested.

I went through it one more time and the only thing that was missing was a `from_params_test` for the `StripeConnectAccount` adapter, so I added that one.